pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -549,6 +551,15 @@ public Object getObject(String type, String value, byte[] byteValue) throws SQLE
       }
     }
 
+    if (type.equals("inet")) {
+      try {
+        return InetAddress.getByName(value);
+      } catch (UnknownHostException e) {
+        throw new PSQLException(GT.tr("IP address of a host could not be determined", type),
```

Comment:
for debugging purposes it would be useful to include the value of the host in the Exception.

Reply:
Thanks for your suggestion and I have changed a little bit.

Reply:
Why not add the hostname instead ?

Reply:
I thought just IP address was enough to show in Exception. If not, I will add hostname.

Reply:
To get hostName from IP Address is like below:
```
InetAddress addr = InetAddress.getByName("192.168.190.62");
String host = addr.getHostName();
System.out.println(host);
```
That's why I use InetAddress.getByName(value).getHostName() in Exception. But when I add that code, it shows me error to add throw declaration again. If I add it, the other files such as PgArray and PgResultset got error because of that declaration. That's why I use value only in Exception.

Reply:
Yes, I do want just the value. If I'm not mistaken the code above has *type* not *value*

Reply:
If so, I have already changed it in my last commit. 
`throw new PSQLException(GT.tr("IP address {0} of a host could not be determined", value)`